### ISSUE TITLE
usage of invalid syntax python 3.7.4

### ISSUE THREAD

**Issue by iTosun** [cyberark/KubiScan#11]
I've been using Kubiscan on python 3.7.4, The first time i tried to use it it gave an exception, about invalid syntax. I've looked trough the file that caused this and found a argument of call_api that uses a python built-in 'async', see following link:

https://github.com/cyberark/KubiScan/blob/master/api/api_client_temp.py#L286

I've changed the argument name which resolved the problem.

**Comment by g3rzi** [comment]
Thank you for reporting !

This is because they changed the python API.
I will add the fix after 6.12.2019 because I have a presentation with demos and I don't want to do any change right now as everything works fine.

After I will add this change I will need to make sure that the [container](https://hub.docker.com/r/cyberark/kubiscan) also works find with this change.

I will update once the fix is ready and the container doesn't have any side affects.

**Comment by disconnect3d** [comment]
Bump; would be nice to have this fixed.

**Comment by g3rzi** [comment]
Thanks @disconnect3d :)
I will update you after testing it together with the container version.

**Comment by g3rzi** [comment]
@iTosun please check now and update if you still have problem.
